### PR TITLE
feat: added more loaders to load_medications

### PR DESCRIPTION
**Pull request by erikperfalk**
- [X] I have battle-tested on Overtaci (RMAPPS1279)
- [X] I have bumped the version as described in the [wiki](https://www.notion.so/Setting-up-Poetry-on-your-local-machine-76647d75e8fa4d9ba553cbc2a49946d9#dca1e0f2ce934a49acd1851aaf882d75)

## Notes for reviewers
Added more data loaders to medications.
Load_prescribed: Changed default option to "False", as this data is unstable and should be reflected upon before used. Maybe you have already discussed it?

### CODE REVIEW

File: src/psycopmlutils/loaders/raw/load_medications.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def load(
                 atc_code matches atc_code_prefix, 0 if not.Defaults to
                 {atc_code_prefix}_value.
             load_prescribed (bool, optional): Whether to load prescriptions. Defaults to
-                True. Beware incomplete until sep 2016.
```
Comment (HLasse) [inline src/psycopmlutils/loaders/raw/load_medications.py:65]: @MartinBernstorff are you using this loader? if so, notice this

File: src/psycopmlutils/loaders/raw/load_medications.py [hunk 2]
```diff
@@ -183,3 +183,194 @@ def antipsychotics(n: Optional[int] = None) -> pd.DataFrame:
...
+    def anti_epileptics(n: Optional[int] = None) -> pd.DataFrame:
+        return LoadMedications.load(
+            atc_code="N03",
+            load_prescribed=False,
+            load_administered=True,
+            wildcard_icd_code=True,
+            n=n,
+        )
+
+    # Data_loader for medications primarly used outside Psychiatry
```
Comment (HLasse) [inline src/psycopmlutils/loaders/raw/load_medications.py:247]:
```suggestion
    # data loaders for medications primarily used outside psychiatry
```

**Review by HLasse — COMMENTED** [review]
LGTM. Wait for Martin to before merging 👍

**Review by HLasse — APPROVED** [review]

File: src/psycopmlutils/loaders/raw/load_medications.py (hunk 1 above)
Comment (MartinBernstorff) [reply src/psycopmlutils/loaders/raw/load_medications.py:65]: Yes, there's a warning added by me as well ;-)

But good point, will have to fix in the future.

**Comment by MartinBernstorff** [comment]
LGTM!

**Pull request merged**